### ISSUE TITLE
JOSE review: classroom use

### ISSUE THREAD

**Issue by tomsing1** [TrevorFrench/R-for-Data-Analysis#17]
The JOSE review guidelines ask 
> Does it describe how it has been used in the classroom or other settings, and how someone might adopt it?

It seems this material is mainly intended for self-study? Or have you (or others) applied it to a classroom setting? If not, how do you envision that might be accomplished?

**Comment by TrevorFrench** [comment]
@tomsing1 Yes, it was intended for self-study. Should I make that more clear in the "Structure of the Book"? I have several anecdotes of it being used in classroom settings; however, I haven't been able to verify the extent/success of those implementations.

**Comment by tomsing1** [comment]
Thank you for clarifying!

**Comment by tomsing1** [comment]
Sorry, reopening this issue after getting feedback from @stats-tgeorge [here](https://github.com/openjournals/jose-reviews/issues/202#issuecomment-1510672845). He liked your suggestion to describe the intended use of this resource in the "Structure of the Book" section.

**Comment by TrevorFrench** [comment]
I added this in PR #24. Let me know if it needs more elaboration. Thanks!
